lib/model.js

```diff
@@ -23,17 +21,13 @@ module.exports = class Model {
     findById(id) {
         return readJSON(`${this.modelName}/${id}`);
     }
-    
```

Comment:
Everything looks great man! Really like how easy your code is to read! Only suggestion is add a space between each function, this is a nit picky way of how i read code and organize though!